recipes-crosswalk/crosswalk/crosswalk_20.50.533.12.bb

```diff
@@ -419,6 +448,7 @@ DEFAULT_CONFIGURATION = "\
 
 # Not available on Yocto, so we use
 # what comes bundled with Chromium.
+# -Duse_system_flac=1
```

Comment:
As my investigation, the root cause is Ostro OS XT removes the `flac`, as https://github.com/ostroproject/ostro-os-xt/blob/master/meta-ostro-xt/recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_%25.bbappend#L1.

The `pkg-config` in bitbake actually takes the right search path defined in `PKG_CONFIG_PATH`. On my system, it points to:
```
/home/nhu/devel/ostro-os-xt/build/tmp-glibc/sysroots/intel-corei7-64/usr/lib/pkgconfig:/home/nhu/devel/ostro-os-xt/build/tmp-glibc/sysroots/intel-corei7-64/usr/share/pkgconfig
```